metecho/api/serializers.py

```diff
@@ -399,7 +427,8 @@ def get_valid_target_directories(self, obj) -> dict:
 
     def validate(self, data):
         if (
-            ScratchOrg.objects.active()
+            not self.instance
```

Comment:
@wlonk This change is to make the validation only happen on initial create?

Reply:
Yep, it is.